app/models/tag.rb

```diff
@@ -1238,7 +1238,7 @@ def unwrangled_query(tag_type, options = {})
       type: tag_type,
       unwrangleable: false,
       wrangled: false,
-      pre_fandom_ids: [self.id]
+      "pre_#{self.type.downcase}_ids": [self.id]
     ))
```

Comment:
We should add `per_page: Tag.per_page` so the limit matches the SQL queries for the other bins.